packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json

```diff
@@ -1,31 +1,31 @@
 {
   "billing_commitment_commit": "Subscribe and get a discount",
   "billing_commitment_commit_again": "Re-subscribe and get a discount",
-  "billing_commitment_none": "You do not currently have a subscription on your {{ serviceName }} service. You can get a discount today if you choose a subscription for your {{ serviceType }}.",
-  "billing_commitment_commit_again_description": "You are currently subscribed to your {{ serviceName }} service until {{ endDate }}. You can continue to get a discount by re-subscribing to your {{ serviceType }}.",
+  "billing_commitment_none": "You do not have a commitment on your {{ serviceName }} subscription. You can get a discount today if you add a commitment to your &lt;dedicated server>{{ serviceType }} subscription.",
```

Comment:
Duplicate #5520.

Reply:
Its not duplicate text. Varun has made a fix to correct invalid text "&lt;dedicated server>"